.github/workflows/ci.yml

```diff
@@ -91,7 +92,7 @@ jobs:
 
       - run: pip install jq yq
 
-      - run: pdm install && pdm run update-deps
+      - run: pdm install && git submodule update --init --recursive && cd @account-abstraction && yarn && yarn compile &&  cd ../spec && yarn && yarn build
```

Comment:
@Vid201 Finally locate the error is here. 

`pdm run update-deps` would interprete into `git submodule update --remote --init --recursive && cd @account-abstraction && yarn && yarn compile &&  cd ../spec && yarn && yarn build`

with `--remote` , submodule would update account-abstraction and bundler-spec repo to latest version.

Reply:
It's weird because nothing changed. I was thinking if the following can be a problem: each test setup geth instance and uses TempDir `test_geth` - after the test is finished this dir is deleted. Can this be a problem if tests are run async?